lib/unwrappr/lock_file_diff.rb

```diff
@@ -63,7 +63,7 @@ def extract_gem_and_change_type(line)
       # '+    websocket-driver (0.6.5)'
       # Careful not to match this (note the wider indent):
       # '+      websocket-extensions (>= 0.1.0)'
-      pattern = /^(?<change_type>[+\-])    (?<gem_name>\S+) \(\d/
+      pattern = /^(?<change_type>[+-])    (?<gem_name>\S+) \(\d/
```

Comment:
A fix for a Rubocop issue: resolve [Style/RedundantRegexpEscape](https://docs.rubocop.org/rubocop/cops_style.html#styleredundantregexpescape).